concepts/derive/about.md

```diff
@@ -1,3 +1,4 @@
 # About
 
 TODO
+- `PartialEq`, `PartialOrd`, `Debug`, `Display`, `Default`
```

Comment:
```suggestion
- `PartialEq`, `Eq`, `PartialOrd`, `Ord`, `Debug`, `Display`, `Default`
```

Probably most of the time we derive partialeq/partialord, it's appropriate to derive the full equality/ordering as well. Obviously they get their own conceptin "total-ordering", but the takeaway here should just be that it's possible to derive these traits.